swedish-oidc-profile.md

```diff
@@ -212,8 +209,66 @@ An OpenID Provider compliant with this profile SHOULD support the PKCE extension
 
 An OpenID Provider MUST NOT allow a Relying Party to use the `plain` code challenge method.
 
+<a name="authentication-request-parameter-extensions"></a>
+### 2.2. Authentication Request Parameter Extensions
+
+This section contains authentication request parameter extensions defined within the scope of this profile.
+
+<a name="client-provided-user-message"></a>
+#### 2.3.1. Client Provided User Message
+
+**Parameter:** `https://claims.oidc.se/1.0/userMessage`
+
+**Description:** When the user message claim is included in an authentication request the issuing client requests
```

Comment:
I may have missed something, but is support for the userMessage request parameter required for an OP in the oidc.se profile? If support for userMessage is OPTIONAL, perhaps we should include an identifier for advertising this capability in an OPs .well-known/openid-configuration ? 

For example:
`{ ... other JSON ...
"https://claims.oidc.se/1.0/userMessage_parameter_supported": true,
  ... or ...
  "id_oidc_se_user_message_parameter_supported": true,
}`

I'm not sure if I've seen "scoped" capability names in openid-configuration before.

Reply:
Yes, its optional. Will add that. Thanks.